### PR TITLE
Decisions query engine

### PR DESCRIPTION
**Pull request by ntietz**
This PR supports all the aggregations we need for this sprint, from [this document](http://www.un.org/en/sc/repertoire/data/sanctions_regimes_graphs.pdf).

Example query:
```
query Count
{
  decisionQuery {
    year,
    count,
    measureType
  }
}
```

This returns:
```
{
  "data": {
    "decisionQuery": [
      {
        "year": 2017,
        "count": 5,
        "measureType": "extend"
      },
      {
        "year": 2017,
        "count": 5,
        "measureType": "limited extend"
      },
      {
        "year": 2016,
        "count": 2,
        "measureType": "establish"
      },
      {
        "year": 2016,
        "count": 5,
        "measureType": "extend"
      },
      ...
```

### CODE REVIEW

**Comment by ntietz** [comment]
@Koleok How would you suggest testing GraphQL stuff? Is there a good library out there for it?

**Comment by Koleok** [comment]
@ntietz [this article](https://medium.com/entria/testing-a-graphql-server-using-jest-4e00d0e4980e) is a good reference for just such a task.

**Comment by Koleok** [comment]
also related to the `inline-import` babel plugin i noted earlier in terms of testing is [`jest-transform-graphql`](https://www.npmjs.com/package/jest-transform-graphql)

File: src/schema/index.js [hunk 1]
```diff
@@ -147,9 +147,11 @@ const mandateSchema = new Schema({
   mandateComponents: [mandateComponentSchema],
 });
 
+const logger = { log: e => console.log(e) };
```
Comment (Koleok) [inline src/schema/index.js:150]: a useful note to highlight the lovely first class function behavior of js:, the logger object here is equiv to 
```js
{ log: console.log }
```

or below on `L154` you could just do `logger: console`
Exception being, if for some reason you **needed** `log` to be explicitly _unary_ instead the _variadic_ version you get by default.

**Review by Koleok — COMMENTED** [review]
forgot to submit this the other day 😱 , I'll add more if i have time tonight but feel free to merge if it is convenient before then.

**Comment by Koleok** [comment]
Oh its already merged 😂 , ignore that review

**Pull request merged**